core/examples/object.rs

```diff
@@ -31,7 +31,8 @@ async fn main() -> Result<()> {
         .try_init();
 
     // Pick a builder and configure it.
-    let builder = services::Memory::default();
+    let mut builder = services::Fs::default();
```

Comment:
Avoid using `fs` as an example, as it may cause unexpected changes in the user's environment.

Reply:
It's for my local testing. I'll roll back it before merging.